dali/core/geom_mat_test.cu

```diff
@@ -19,7 +19,7 @@
 
 namespace dali {
 
-TEST(Mat, Constructor) {
+TEST(MatTest, Constructor) {
```

Comment:
What else can it be? It looks like renaming John, Bob and Alice to JohnHuman, BobHumand and AliceHuman.

Reply:
Naming test suites in a way, that it ends with `Test` seems to be quite a common thing. Be aware, that for a test case `TEST(Mat, Constructor)` gtest's macro-magic generates a class `Mat_Constructor`. In no case such name implies, that this class is actually a test. I'd say it's a thing we could add to the code style

#1330

Reply:
I strongly oppose making it a requirement. While I may even agree upon this in case of something as short as "Mat", adding "Test" to all test names will just add noise and produce long strings in GTest output.